data/id/legislature/Daniel-D-Foreman-c8f4817a-72df-4164-a5fb-4b56ad57d787.yml

```diff
@@ -0,0 +1,22 @@
+id: ocd-person/c8f4817a-72df-4164-a5fb-4b56ad57d787
+name: Daniel D. Foreman
+email: DForeman@senate.idaho.gov
+image: https://legislature.idaho.gov/wp-content/uploads/sessioninfo/2023/directory/Foreman4269.jpg
+party:
+- name: Republican
+roles:
+- start_date: 2022-12-01
+  type: upper
```

Comment:
@bfossen-ce this person served in the Senate district 5 from 12/01/2016 through 12/01/2018.